### ISSUE TITLE
Layered Navigation not working - nothing shown in frontend

### ISSUE THREAD

**Issue by chequille** [magento/magento2#14995]
### Preconditions
1. Magento 2.2.4
2. PHP 7.0.29
3. MySQL 5.6.40

### Expected result
1. At least Price should be shown as Layered Navigation 

### Actual result
1. Nothing shown

### Addtional informatione
In the template .../layer/view.phtml the function $block->canShowBlock() returns alrady false, therefore nothing else is executed.
catalog is an anchor, and the simple products in this category do have a different price.
So it should work normally
Any idea?

BR
Chequille

**Comment by chequille** [comment]
Hi,
I found the solution for this issue.
I always thought, that the command magento cache:clean is cleaning all cached data. But this is not the case.
I got the layered navigation working again after I selected in the Backend->Cache Management 

1. - Flush Cache Storgage
2. - Flush Magento Cache
3. - Flush Static FIles Cache

After this was done, layered navigation is shown again.

Therefore, only a question left: is there any other command which is doing this job in the command line?

BR
Chequille
